docs/METRICS.md

```diff
@@ -1,14 +1,14 @@
-## Page Shot Metrics
+## Firefox Screenshots Metrics
 
-A summary of the metrics Page Shot will record, and what we're looking for in those metrics.
+A summary of the metrics Screenshots will record, and what we're looking for in those metrics.
```

Comment:
I think "Firefox Screenshots" would be clearer than "Screenshots" in some spots in this prose-heavy part (up to the 'usage metrics' section).